src/helm/proxy/clients/model_critique_client.py

```diff
@@ -0,0 +1,203 @@
+        return prompt
+
+    def _task_to_requests(self, task: CritiqueTaskTemplate, fields: Dict[str, str]) -> List[Request]:
+        base_prompt: str = self._interpolate_fields(task.instructions, fields)
+
+        requests: List[Request] = []
+        for question in task.questions:
+            prompt: str = base_prompt + "\n\n" + self._question_to_prompt(question, fields)
+            if question.question_type == "free_response":
+                # TODO: Make max_tokens configurable
+                max_tokens = 100
+            elif question.question_type == "checkbox":
+                # We multiply by 2 because the model will generate a comma after each option.
+                max_tokens = len(question.options) * 2
+            else:
+                max_tokens = 1
+            request = Request(
+                model=self._model_name,
+                prompt=prompt,
+                max_tokens=max_tokens,
+                echo_prompt=False,
+            )
+            requests.append(request)
+        return requests
+
+    def _execute_requests(self, requests: List[Request], num_respondents: int) -> List[List[RequestResult]]:
+        """Execute a list of requests and return the responses."""
+        responses: List[List[RequestResult]] = []
+        for request in requests:
+            responses.append([])
+            for i in range(num_respondents):
+                request_with_random = dataclasses.replace(request, random=str(i))
+                response: RequestResult = self._client.make_request(request_with_random)
+                responses[-1].append(response)
+        return responses
+
+    def _parse_completion_to_question_choice(self, completion: str) -> List[str]:
+        """Convert a completion to a list of answer represented by a list of capital letters."""
+        completion_parsed = completion.replace(" ", "").replace("\n", "").replace(".", "").upper()
+        answers = completion_parsed.split(",")
+        if len(answers) < 1:
+            raise CritiqueParseError(f"Invalid answer: {completion}. There are no answers once parsed: {answers}.")
+        for i, answer in enumerate(answers):
+            if answer not in string.ascii_uppercase:
+                raise CritiqueParseError(
+                    f"Invalid answer: {completion}. Some answers are not capital letters, once parsed: {answers}. "
+                    f"Error happened at answer {i}, which is {answer}."
+                )
+        return answers
+
+    def _multiple_choice_completion_to_answer(
+        self, question: CritiqueQuestionTemplate, completion: Sequence
+    ) -> Optional[str]:
+        """Convert a multiple choice completion to an answer."""
+        assert question.question_type == "multiple_choice"
+        try:
+            answers: List[str] = self._parse_completion_to_question_choice(completion.text)
+            assert len(answers) == 1, f"Invalid answer: {completion}. Multiple choice questions should have one answer."
```

Comment:
Should be a `CritiqueParseError`. (i.e. you tried to parse out one answer, but there were multiple answers)